javascript/javascript1/week3/warmup1.js

```diff
@@ -0,0 +1,22 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+
+function removeName(arr,name){
```

Comment:
Nice and concise; good job! ⭐ (Small nitpick: you are technically not using `nameToRemove`!)